web/src/tippyjs.ts

```diff
@@ -230,6 +230,21 @@ export function initialize(): void {
             return false;
         },
     });
+    delegate("body", {
+        target: "#add-todo-modal .dialog_submit_button_container",
+        appendTo: () => document.body,
+        onShow(instance) {
+            const content = $t({defaultMessage: "Please enter a title."});
+            const $elem = $(instance.reference);
+            // Show tooltip to enter question only if submit button is disabled
+            // (due to question field being empty).
```

Comment:
This comment is for the poll modal... Also, is this entire block of code even needed since titles are optional?